### ISSUE TITLE
After I trained on my own 3D data and predicted on the test set, I found some samples with null predictions

### ISSUE THREAD

**Issue by maolimin** [bowang-lab/U-Mamba#77]
"metrics": {
                "1": {
                    "Dice": 0.0,
                    "FN": 353,
                    "FP": 0,
                    "IoU": 0.0,
                    "TN": 22478495,
                    "TP": 0,
                    "n_pred": 0,
                    "n_ref": 353
                }
            },
            "prediction_file": "/data0/maolimin/U-Mamba/data/nnUNet_results/Dataset202_Breast/nnUNetTrainerUMambaBot__nnUNetPlans__3d_fullres/fold_0/validation/br_130.nii.gz",
            "reference_file": "/data0/maolimin/U-Mamba/data/nnUNet_preprocessed/Dataset202_Breast/gt_segmentations/br_130.nii.gz"

There are also cases where the inference is null on the validation set,Looking forward to your reply
